docs/best_practice/crate_ami.txt

```diff
@@ -133,13 +124,14 @@ configuration is used when a cluster with 3 or more nodes is set up.
 Launch AMI by using the AWS CLI
 -------------------------------
 
-The following example shows how to run a single instance of type
-``m3.medium`` and pass the AWS credentials as a file (provided as
-``user-data.sh``).
+The following example shows how to run a single instance of type ``m3.medium``. 
+The instances are assigned with the IAM role ``crate-ec2`` which allows the 
```

Comment:
Instances which have the IAM role ``crate-ec2`` assigned will securely fetch their credentials which are then used for authentication in the discovery mechanism.
The following example shows how to run a single instance of type ``m3.medium``. The adapted Crate configuration is 
+passed via ``user-data.sh`` file.